spec/install/gemspecs_spec.rb

```diff
@@ -63,6 +63,33 @@
     expect(out).to include("Bundle complete!")
   end
 
+  it "reads gemspecs respecting their encoding" do
+    skip "Too cool for 1.8" if RUBY_VERSION < "1.9"
```

Comment:
This is not informative, please use a comment that explains why we need to skip 1.8 in this test.

Reply:
What about "unicode was probably not fully supported on 1.8"?

Reply:
Basically the reason is that we have decided it's not worth even trying to make this work on 1.8. We're not sure it's actually supported and blind guessing from Travis logs feedback is... tough.

Reply:
Pushed a new skip message, hopefully more serious and informative :)